src/parse/cast.jl

```diff
@@ -123,29 +123,29 @@ function parse_kwargs(def)
 
     # (name, type, isflag)
     for each in def[:kwargs]
-        push!(types.args, Expr(:tuple, to_option_or_flag(each)...))
+        push!(types.args, to_option_or_flag(each))
```

Comment:
The most controversial part.

Without this change, default value is not parsed as expression if it is given in that way.